src/status_im/navigation/view.cljs

```diff
@@ -52,12 +52,18 @@
 
 (defn screen
   [screen-key]
+
   (reagent.core/reactify-component
    (fn []
-     (let [{:keys [component options]}   (get (if js/goog.DEBUG
+     (let [screen-details                (get (if js/goog.DEBUG
                                                 (get-screens)
                                                 screens)
                                               (keyword screen-key))
+           qualified-screen-details      (get (if js/goog.DEBUG
+                                                (get-screens)
+                                                screens)
+                                              (keyword "screen" screen-key))
```

Comment:
Hi @J-Son89, Please can you elaborate why this is needed?

Reply:
For sure, 
As far as I understand it,  when we navigate we use the `name` function to pass the screen name to the navigation package. `name` removes the qualification and so we have to add the `:screen/` qualification back here. 
It's not such a change as in either case we need this `keyword` processing here anyway but we need to support both until we would have completely migrated `keywords` over.

Reply:
there could be some other approaches to handling this too.

Reply:
the name doesn't remove this qualification, this was done by the `(clj->js)` function. It seems that these keyword qualifications
 are not playing well with JavaScript.

I think we should be a little careful to use them until we figure out a solution for this. Because we are covering this specific case, we might miss some other places and cause unintentional bugs.

Reply:
perhaps not in this case, but it does:
https://cljs.github.io/api/cljs.core/name
<img width="212" alt="Screenshot 2024-02-15 at 13 29 49" src="https://github.com/status-im/status-mobile/assets/22799766/2650c10d-27ff-4acd-8f25-51d07e43ce16">

Not so sure, from what I seen this is the only place this happens. Will investigate more as it's a worthy concern and maybe it can be handled cleaner. In this particular case we still have to be careful to modify the keywords from a string anyway.

Reply:
Oh sorry, I thought you meant `:name` parameter is getting stripped inside the navigation package (as that's what we are using while navigating).

Anyway, thank you for pointing out that the `name` function also has `clj->js` limitations while working with js.

Reply:
Ah okay, gotcha. In any case I'll do a further check to ensure there's no too many vulnerabilities related to this.

Reply:
If this works for something, we can preserve the namespace by extracting `fqn` from a keyword. E.g.

`(.-fqn :my/keyword)`
;; => `"my/keyword"`